### ISSUE TITLE
Error message when running Migmap with -R set to TRA,TRB,TRG,TRD

### ISSUE THREAD

**Issue by Zoites** [antigenomics/arda#50]
I'm getting the following when running Migmap with -R set to TRA,TRB,TRG,TRD:
java.lang.IllegalArgumentException: multiply() should be called with a number of 0 or greater not: -91
[StackTrace-Short]

I have attached error log file. 
[_migmap_error.log](https://github.com/mikessh/migmap/files/1697229/_migmap_error.log)

Has anyone seen this before? It is a bug?

**Comment by gnetsanet** [comment]
I, too, ran into when I turn off all the filters ( except the quality) as below.

`migmap --allow-incomplete --allow-no-cdr3 --allow-noncanonical --allow-noncoding --unmapped unmapped.fastq --data-dir /home/ngebremedhin/miniconda2/share/igblast  -S human -R TRA,TRB,TRG,TRD --report ./migmap_report.txt in.fq ./output.txt`

**Comment by LukeHebert** [comment]
Yes, I get this too. I used "-R IGH,IGK,IGL" but still received the error: 

Internal: java.lang.IllegalArgumentException: multiply() should be called with a number of 0 or greater not: -170. See _migmap_error.log for details.
